pallets/subtensor/src/registration.rs

```diff
@@ -317,7 +317,7 @@ impl<T: Config> Pallet<T> {
 
         // --- 3. Ensure the passed network allows registrations.
         ensure!(
-            Self::if_subnet_allows_registration(netuid),
+            Self::get_network_pow_registration_allowed(netuid),
```

Comment:
is there some idiomatic reason for this name change?
I thjink the previous one has a good naming, maybe something like.... `Self::subnet_allows_registrations(netuid)` which could fit with the context and its semantics. What do you think?

Reply:
if_subnet_allows_registration was a duplicate of another function which was already implemented.